# Signals assertion/retraction of a triple.
# Note: Triples as such are considered immutable, hence there cannot be an
# 'update' of a triple.

Triple data

int8 ADDED=1
int8 REMOVED=3

int8 updateType
